cmake/fuzzing_options.cmake

```diff
@@ -16,8 +16,12 @@
 
 include(compiler_id)
 
-option(FUZZING "Build for local Fuzz Testing" OFF)
-option(OSS_FUZZ "Build for OSS Fuzz Environment" OFF)
+option(FUZZING "Build for Fuzz Testing (local fuzzing and OSS Fuzz)" OFF)
+
+# Assume OSS Fuzz if LIB_FUZZING_ENGINE environment variable is set.
+if(DEFINED ENV{LIB_FUZZING_ENGINE})
+  set(OSS_FUZZ ON)
```

Comment:
`OSS_FUZZ` no longer seems to be in use aside from the check below. Maybe just remove it?

Reply:
Removed. I just thought it is more readable.